src/javascript/app/pages/resources/economic_calendar/economic_calendar.js

```diff
@@ -26,7 +26,7 @@ const EconomicCalendar = (() => {
         if (loader) loader.remove();
 
         BinarySocket.wait('website_status', 'authorize', 'landing_company').then(() => {
-            $('.calendar-footer').setVisibility(isEuCountry());
+            $('.calendar-footer').setVisibility(true);
         });
```

Comment:
please check if we can remove all this part and also remove `invisible` class from the template